.travis.yml

```diff
@@ -20,15 +21,30 @@ cache:
 
 install:
     - mkdir -p build/logs
-    - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ~/xdebug.ini || return 0
+    - mv ${HOME}/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini ~/xdebug.ini || return 0
     - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction
     - if [ "$CHECKS" = "yes" ]; then travis_retry composer install-devtools; fi;
 
 script:
     - cp ~/xdebug.ini ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini || return 0
     - phpunit
-    - rm ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini || return 0
+    - rm ${HOME}/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini || return 0
```

Comment:
Tilde (~) is shell's feature. It will likely work with any shells, but envvar is more portable in theory.